contrib/sort.py

```diff
@@ -84,22 +96,30 @@ def fix_profile(filename):
 
 
 def main(args):
+    if len(args) < 1:
+        print(__doc__, file=stderr)
+        return
```

Comment:
Add an exit code.

Reply:
@rusty-snake [on Oct 19](https://github.com/netblue30/firejail/pull/5429#discussion_r999637842):

> Add an exit code.

Done.

(Sorry for the delay, I'm having some infrastructure issues)